relay-server/src/metrics_extraction/event.rs

```diff
@@ -543,7 +508,15 @@ mod tests {
         )
         .unwrap();
 
-        let metrics = extract_metrics(event.value_mut().as_mut().unwrap());
+        // Create a project config with the relevant feature flag. Sanitize to fill defaults.
+        let mut project = ProjectConfig {
+            features: [Feature::SpanMetricsExtraction].into_iter().collect(),
+            ..ProjectConfig::default()
+        };
+        project.sanitize();
```

Comment:
Note how we also test that feature flag + sanitize + extraction all works together.